spec/peep_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'peep'
+require 'database_helpers'
+require 'users'
+
+describe Peep do
+  before do
+    truncates
+    add_test_users
+  end
+
+  it 'creates a new peep' do
+    truncates
+    user = Users.create(name: 'test name', email: 'test@email.com', password: 'pword123', username: 'test_username')
```

Comment:
Could this be doubled rather than hard coded to allow autonomy of tests?

Reply:
I wish I understood doubles - but yes I agree.